docs/b2b-edition/specs/api-v3/company/company.yaml

```
@@ -38,7 +38,7 @@ info:
     
     ![Graphic explainer for company account hierarchies](https://storage.googleapis.com/bigcommerce-production-dev-center/images/account-hierarchies-examples.png)
 
-    **NOTE:** This functionality is available upon request for stores using the Buyer Portal experience and Independent Companies behavior. You can request Account Hierarchy by contacting our support team.
+    **NOTE:** This functionality is enabled by default on all new B2B Edition stores. It is also available upon request for existing stores using the Buyer Portal experience and Independent Companies behavior. You can request Account Hierarchy by contacting our support team.
```

Can you remove the passive voice? Maybe you can use something like the following:
```suggestion
    **NOTE:** This functionality comes enabled by default on all new B2B Edition stores. It is also available upon request for existing stores using the Buyer Portal experience and Independent Companies behavior. You can request Account Hierarchy by contacting our support team.
```